README.md

```diff
@@ -39,6 +39,14 @@ Checks before sending a PR:
 - Run all tests to assure nothing else was accidentally broken. This is done by running: `npm test`.
 - Make sure you have added the necessary tests and [documentation](http://usejsdoc.org/) for your changes.
 
+## Development
```

Comment:
It took me a minute to figure out how to do dev, so I added this to help people. Please correct me if there's a faster way to iterate.

Reply:
nice!